demo/index.html

```diff
@@ -0,0 +1,63 @@
+<!doctype html>
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<html>
+<head>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+
+  <title>iron-input demo</title>
+
+  <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+
+  <link rel="import" href="../iron-input.html">
+
+  <link rel="stylesheet" href="../../paper-styles/demo.css">
+
+</head>
+<body>
+
+  <template is="x-autobind">
+
+    <section>
+
+      <p>
+        <input is="iron-input" bind-value="{{bindValue}}" value="{{value::input}}">
+        <br>
+        bind to <code>bind-value</code>: <span>[[bindValue]]</span>
+        <br>
+        bind to <code>value::input</code>: <span>{{value}}</span>
+      </p>
+
+      <p on-click="setValue">
+        set bind-value to: <input> <button value="bindValue">set</button>
+        <br>
+        set value to: <input> <button value="value">set</button>
+      </p>
+
+    </section>
+
+  </template>
+
+  <script>
+
+    var scope = document.querySelector('template[is=x-autobind]');
+
+    scope.setValue = function(event) {
```

Comment:
Just curious: was this actually working for you with `rc.6`? I @blasten was trying this yesterday and it didn't seem to work (poking x-autobind with a handler).